### DISCUSSION TITLE
Bot not working

### DISCUSSION THREAD

**Discussion by oshhin** [aa-sikkkk/FortuneTeller#2] [Q&A] [answered]
I am trying to run the bot but When ever I try to run the script it shows error in the terminal.. (mainly import errors) help me fix the issue

**Comment by aa-sikkkk** [comment, marked as answer]
> I am trying to run the bot but When ever I try to run the script it shows error in the terminal.. (mainly import errors) help me fix the issue

To tackle the problem, consider running the script in a Virtual Environment using Docker and utilizing Remote WSL. This approach can streamline your workflow and ensure compatibility. Make sure to carefully read the guide and install all necessary dependencies, including setting up your own Telegram bot API. If you need more help, just give a shout!

**Reply by oshhin** [reply]
It worked, thanks for the instant reply..

**Reply by JEAZY-LEKE** [reply]
i think a video should be made concerning this because ive been on if for days now

**Reply by aa-sikkkk** [reply]
> i think a video should be made concerning this because ive been on if for days now
> 
> 

https://youtu.be/O3DwM6hLsQI?si=D199mH8S38lgSbU6
You can see this for reference.

**Reply by samyambhandari** [reply]
I need help for setup
